easybuild/easyconfigs/d/DualSPHysics/DualSPHysics-5.0.233.patch

```diff
@@ -0,0 +1,25 @@
+diff -Naur DualSPHysics-5.0.175/src/source/CMakeLists.txt DualSPHysics-5.0.175/src/source/CMakeLists.txt
```

Comment:
Please add a description of what the patch does and author info.
Also the file name of the patch should be descriptive.

And since this patch touches on gencode, that info should preferably be taken from the cuda_compute_capabilities info available from easybuild if possible.